policy-report/api/v1alpha1/policyreport_types.go

```diff
@@ -45,15 +45,23 @@ type PolicyReportSummary struct {
 	Skip int `json:"skip"`
 }
 
-// PolicyStatus has one of the following values:
-//   - pass: indicates that the policy requirements are met
-//   - fail: indicates that the policy requirements are not met
-//   - warn: indicates that the policy requirements and not met, and the policy is not scored
-//   - error: indicates that the policy could not be evaluated
-//   - skip: indicates that the policy was not selected based on user inputs or applicability
+// PolicyResult has one of the following values:
+//   - pass: the policy requirements are met
+//   - partial-pass: the policy requirements are met but requires manual assessment
```

Comment:
@JimBugwadia I'm not recalling (or wasn't involved with) the use case for this. wouldn't this just be "warn"?